<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineQuery">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>711</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Beyond bags of features: Spatial pyramid matching for recognizing natural scene categories</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSingleSearch">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Search</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainSingleSearch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>711</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushCitations">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse Citations</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%v of %m</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushShowPage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Page</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSaveCurrent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Current</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushScholarInfo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSource">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Google Scholar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IEEE Xplore</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushReferences">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>460</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Reference</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
